draft-ietf-mls-protocol.md

```diff
@@ -916,6 +916,19 @@ enum {
 opaque SignaturePublicKey<1..2^16-1>;
 ~~~~~
 
+Note that each new credential that has not already been validated
+by the application SHOULD be validated against the Authentication
+Service.
```

Comment:
maybe just "MUST be validated."

Reply:
I don't think you can make this a MUST.  You can imagine applications that don't really care about authentication, but want all the other properties.  Also, there is no comparable requirement in TLS.